lib/svgo/css-style-declaration.js

```diff
@@ -0,0 +1,79 @@
+'use strict';
+
+var csstree  = require('css-tree'),
+    csstools = require('../css-tools');
+
+var CSSStyleDeclaration = function(node) {
+  this.parentNode = node;
+
+  this.properties = new Map();
+  this.length     = 0;
+
+  this.update(); // initialize with styles from style attribute
+};
+
+
+CSSStyleDeclaration.prototype.update = function() {
+  var self = this;
+  if(!this.parentNode.hasAttr('style')) {
+    return;
+  }
+  var inlineCssStr = this.parentNode.attr('style').value;
+  var cssoDeclarations = csstree.parse(inlineCssStr, {context: 'declarationList'});
```

Comment:
Since value details is not needed here, we can use `parseValue: false` option for `parse()`. It can improve performance a little.